cfgov/cfgov/settings/base.py

```diff
@@ -152,7 +152,8 @@
             V1_TEMPLATE_ROOT,
             V1_TEMPLATE_ROOT.child('_includes'),
             V1_TEMPLATE_ROOT.child('_layouts'),
-            PROJECT_ROOT.child('static_built')
+            PROJECT_ROOT.child('static_built'),
+            PROJECT_ROOT.child('wellbeing').child('templates'),
```

Comment:
I remember that we discussed this, and this works for me, but I do wonder about having this folder included in both template backends (here, for Jinja, but also above, as part of the Django templates because of `'APP_DIRS': True`). Your templates appear to use Jinja, and they seem to work, so I wonder if it just takes the last template engine defined?